code/modules/recycling/disposal/bin.dm

```diff
@@ -234,16 +229,10 @@
 	qdel(H)
 
 /obj/machinery/disposal/deconstruct(disassembled = TRUE)
-	var/turf/T = loc
 	if(!(flags_1 & NODECONSTRUCT_1))
-		if(stored)
-			stored.forceMove(T)
-			src.transfer_fingerprints_to(stored)
-			stored.anchored = FALSE
-			stored.density = TRUE
-			stored.update_icon()
+		new /obj/structure/disposalconstruct(loc, null, SOUTH, FALSE, src)
 	for(var/atom/movable/AM in src) //out, out, darned crowbar!
-		AM.forceMove(T)
+		AM.forceMove(loc)
```

Comment:
`get_turf(src)` is the standard for emptying forceMoves, although this will work 99.9% of the time